src/components/DataViewer/DataViewer.tsx

```diff
@@ -15,45 +17,70 @@ const StyledModal = styled(Modal)`
     left: 50%;
     transform: translate(-50%, -50%);
     border: 0;
-    border-radius: 5px;
+    border-radius: 2em;
     outline: none;
     padding: 0;
     width: 80%;
     max-height: 80%;
-    background: rgb(255, 255, 255);
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
     box-shadow: rgb(200, 200, 200) 0 2px 10px;
     overflow: hidden;
+    max-width: 1000px;
 `;
-const Header = styled.h3`
+
+const Header = styled.div`
     display: flex;
     justify-content: space-between;
     align-items: center;
-    background-color: #eeeeee;
-    padding: 0.5rem;
-    margin-top: 0;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
```

Comment:
Is this opacity level something that will be reused? (e.g. whenever we use a modal). If it is, do you think we should add this to `themes`? (e.g. as `blurredBackgroundColor`)

Reply:
Yeah, I agree on making these colours reusable; although, I was expecting this to be changed during a re-haul of the theming implementation. But, if you think it's best that it's done just now then that can be done

Reply:
Also I think rather than adding the opacity as a theme, we should make new colors that looks the same but with 100% opacity. It is because opacity changes the coloring by making it more transparent, and in dark mode it doesn't work as the background it black, it just makes things less visible